### PR TITLE
Deleted backend service from cosv

### PR DESCRIPTION
**Pull request by Cheshiriks**

### CODE REVIEW

**Comment by kgevorkyan** [comment]
why its so many duplicate ci jobs?

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt [hunk 1]
```diff
@@ -33,4 +35,58 @@ interface OrganizationRepository {
...
     )
     fun getOrganizationByName(@Param("name") name: String): Organization
+
+    /**
+     * @param userId
+     * @param organizationId
+     * @return found [Role]
+     */
+    @Query(
+        value = "select role from save_cloud.lnk_user_organization where user_id = :user_id and organization_id = :organization_id",
```
Comment (kgevorkyan) [inline save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt:45]: why we need native quires here?

Comment (akuleshov7) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt:45]: two different data sources in one app and Spring cannot work with it well

**Review by akuleshov7 — APPROVED** [review]

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt (hunk 1 above)
Comment (akuleshov7) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt:45]: but may be Vlad can use this: https://www.baeldung.com/spring-boot-configure-multiple-datasources

Comment (Cheshiriks) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/OrganizationRepository.kt:45]: It's like that. In the next few PR I'll try to do it a differently

**Pull request merged**